cpp/util/libevent_wrapper.cc

```diff
@@ -264,8 +284,24 @@ void HttpServer::Bind(const char* address, ev_uint16_t port) {
 }
 
 
+void StatsHandlerInterceptor(evhttp_request* req,
+                             const HttpServer::HandlerCallback& cb) {
+  const auto start(system_clock::now());
+  cb(req);
+  const auto duration(system_clock::now() - start);
```

Comment:
Why not make this ```milliseconds``` right away, and avoid the cast below? The subtraction should be done at the full precision, right? Or does this cause some compile time whining, because the conversion is lossy?